src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -135,7 +136,11 @@ class DocBaseViewer extends BaseViewer {
             this.thumbnailsSidebarEl = document.createElement('div');
             this.thumbnailsSidebarEl.className = `${CLASS_BOX_PREVIEW_THUMBNAILS_CONTAINER}`;
             this.thumbnailsSidebarEl.setAttribute('data-testid', 'thumbnails-sidebar');
-            this.containerEl.parentNode.insertBefore(this.thumbnailsSidebarEl, this.containerEl);
+            this.rootEl.insertBefore(this.thumbnailsSidebarEl, this.containerEl);
+
+            if (this.getCachedThumbnailsToggledState()) {
```

Comment:
Should this call live in `initThumbnails`, since the same value is used there as well?

Reply:
The thinking behind this was to add the thumbnails open class as soon as possible so that the layout of the viewer is set while the pdf gets rendered. If we wait until `initThumbnails` to add the open class, then the UI will jump a little after the PDF content is fetched which is what design wanted to avoid